build-pipeline/windows-pipeline.groovy

```diff
@@ -0,0 +1,17 @@
+@Library('dotnet-ci') _
+
+// TODO: how to do this?
+//
+// if (params.RS3) {
+//   label('windows.10.amd64.serverrs3.open')
```

Comment:
@mmitche / @MichaelSimons - what's the pipelines equivalent for this? https://github.com/dotnet/dotnet-docker/blob/master/netci.groovy#L34-L39

Reply:
simpleNode('windows.10.amd64.serverrs3.open') will do the same thing